src/style.css

```diff
@@ -1 +1,140 @@
-/* empty source */
+/* #region page */
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+@font-face {
+  font-family: "Roboto", sans-serif;
+}
+
+/* #endregion */
+/* #region card */
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 198px;
+  border: solid #f3f3f3 1px;
+  border-radius: 5px;
+  background-color: #fff;
+}
+
+.card__item-macspecs {
+  padding-top: 33px;
+  width: 160px;
+  height: 134px;
+  padding-left: calc(50% - 160px / 2);
+}
+
+.card__item-macspecs-text {
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  line-height: 18px;
+  padding: 17px;
+  padding-bottom: 4px;
+}
+
+.card__item-prdcode {
+  display: flex;
+  align-items: center;
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  padding: 0.01px 17px;
+  color: #616070;
+}
+/* #endregion */
+/* #region stars */
+.card__item-stars {
+  display: flex;
+  list-style: none;
+  padding: 16px;
+}
+
+.stars {
+  display: flex;
+  list-style: none;
+  padding: 16px;
+}

```

Comment:
1. Same styles
2. I don't see using of `card__item-stars` class in your html, so I suggest to remove it